deid/dicom/actions.py

```diff
@@ -23,47 +23,36 @@
 """
 
 from deid.logger import bot
-from deid.dicom.tags import update_tag
 from deid.utils import get_timestamp
 
-import re
-
-
 # Timestamps
 
 
-def jitter_timestamp(dicom, field, value):
+def jitter_timestamp(field, value):
     """if present, jitter a timestamp in dicom
     field "field" by number of days specified by "value"
     The value can be positive or negative.
 
     Parameters
     ==========
-    dicom: the pydicom Dataset
     field: the field with the timestamp
     value: number of days to jitter by. Jitter bug!
 
     """
     if not isinstance(value, int):
         value = int(value)
 
-    original = dicom.get(field)
+    original = field.element.value
+    new_value = original
 
     if original is not None:
 
         # Create default for new value
         new_value = None
-
-        # If we have a string, we need to create a DataElement
```

Comment:
hmm I think since we just pass field as is, it could still be a string?

Reply:
I don't believe so.  By the time we're performing this (via _run_action()), the string representation of the field has been evaluated to be a DicomField object by find_by_values() or expand_field_expression().   When parser.perform_action() calls _run_action(), we're iterating over a list of DicomFields.